### ISSUE TITLE
Cherry sled texture is missing

### ISSUE THREAD

**Issue by SilentSimulacrum** [MehVahdJukaar/SnowySpirit#53]
The texture for the cherry wood sled is missing.

Mod Version: 2.2.1
Forge Version: 47.1.0
Minecraft Version: 1.20.1

**Comment by NoobiusMcGuffin** [comment]
This is also on fabric, including the textures of modded sleds

**Comment by MehVahdJukaar** [comment]
Weirdly I can't replicate in dev

**Comment by SilentSimulacrum** [comment]
After peeking into the Jar downloaded from curseforge the texture is just not there. 
![Q8flbun](https://github.com/MehVahdJukaar/SnowySpirit/assets/105311418/6a207984-2d55-435b-beed-9313cc229777)

Maybe not included when built?

**Comment by linkovich2** [comment]
Seeing this in 1.19.2 forge as well. All modded wood sleds are not seeming to find the correlated texture. For example:
```
[10:54:16] [Render thread/WARN]: Failed to load texture: snowyspirit:textures/entity/sled/quark/bamboo.png
java.io.FileNotFoundException: snowyspirit:textures/entity/sled/quark/bamboo.png
	at net.minecraft.server.packs.resources.ResourceProvider.m_215590_(ResourceProvider.java:20) ~[client-1.19.2-20220805.130853-srg.jar%23629!/:?]
	at java.util.Optional.orElseThrow(Optional.java:403) ~[?:?]
	...
```

Or 

```
[10:50:00] [Render thread/WARN]: Failed to load texture: snowyspirit:textures/entity/sled/atmospheric/kousa.png
java.io.FileNotFoundException: snowyspirit:textures/entity/sled/atmospheric/kousa.png
	at net.minecraft.server.packs.resources.ResourceProvider.m_215590_(ResourceProvider.java:20) ~[client-1.19.2-20220805.130853-srg.jar%23629!/:?]
	at java.util.Optional.orElseThrow(Optional.java:403) ~[?:?]
	...
```

Edit: I do have the dynamic resource pack enabled

**Comment by MehVahdJukaar** [comment]
fixed

**Comment by linkovich2** [comment]
I know you're going on hiatus, but when you get back any chance this fix can come to the 1.19.2 branch as well? https://github.com/MehVahdJukaar/SnowySpirit/commit/72648138cd85a6054955da518a23d7d4c3e8c465

**Comment by linkovich2** [comment]
To be more specific, it seems like the only change required is this one: https://github.com/MehVahdJukaar/SnowySpirit/commit/72648138cd85a6054955da518a23d7d4c3e8c465#diff-235a3d8b7e2ff314a7dc262f11fa0bad1a5686274c8206a128ad9d465c785ed3

However, I can't build the 1.19.2 version to test because it can't pull moonlight 1.19.2-2.3.12 for whatever reason
